docs/design/scheduler.md

```diff
@@ -0,0 +1,46 @@
+
+# The Kubernetes Scheduler
+
+The Kubernetes scheduler runs as a process alongside the other master
+components such as the API server. Its interface to the API server is to watch
+for Pods with an empty PodSpec.NodeName, and for each them to post a Binding
```

Comment:
"Its interface to the API server is to watch for Pods with an empty PodSpec.NodeName, and for each pod, it posts a Binding indicating where the Pod should be scheduled."